src/Jenssegers/Date/Date.php

```diff
@@ -47,6 +47,42 @@ public static function make($time = null, $timezone = null)
     {
         return static::parse($time, $timezone);
     }
+    
+    /**
+     * Create new Date object from format and localized date
+     *
+     * @param  string  format
+     * @param  string  $date
+     * @return Date
+     */
+    public static function createFromFormat($format, $date, $tz = null)
+    {
+        // Get translator
+        $lang = new Translator;
```

Comment:
Can't you just get the current static::getTranslator() instead of creating a new one?